cgi-bin/DW/Hooks/EmbedWhitelist.pm

```diff
@@ -105,6 +105,8 @@ my %host_path_match = (
 
     "codepen.io"            => [ qr!^/enxaneta/embed/!, 1 ]
```

Comment:
It turns out this fails to compile because the line adding codepen.io doesn't end with a comma.